services/app-graphql/src/queries/indexContracts.graphql

```diff
@@ -87,20 +87,7 @@ fragment rateRevisionFragmentForIndexContract on RateRevision {
     formData {
         rateType
         rateCapitationType
-        rateDocuments {
```

Comment:
👍 this makes sense, and I'll update the mockIndexContract to account for this. I think I'll let this merge first and then push my changes